### PR TITLE
Updating readme

### PR DESCRIPTION
**Pull request by morsh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,47 @@
...
+3. Run through the **Application Insights** setup and fill in **API Key** and **Application ID** according to the application insights account associated with your registered bot.
 
-There are 3 ways to deploy to Azure:
+### Development
+
+```bash
+yarn start:dev
+```
 
+1. Open **http://localhost:3000**
```
Comment (eladiw) [inline README.md]: ibex + graphql/ apollo reference here?

Comment (morsh) [inline README.md]: why?

Comment (eladiw) [inline README.md]: didn't we want a cross reference between the projects?

**Comment by morsh** [comment]
Yes, can you add a last section in the readme file referencing the repo with GQL?

~MoЯ

**Pull request merged**